### ISSUE TITLE
empty output, error in debug log

### ISSUE THREAD

**Issue by jDmacD** [loafoe/terraform-provider-ssh#65]
version: 2.5.0
terrform host: alpine wsl
target host: ubuntu

* Connection is successful
* The file is being run 
* `/tmp/fooisbar` is being created

## TF Snippet
```yaml
resource "ssh_resource" "k3s_coord_server_config" {
  when = "create"
  triggers = {
    always_run = "${timestamp()}"
  }

  host         = module.k3s_coord_server.k3s_node_name
  user         = "local"
  private_key  = file(pathexpand("~/.ssh/cloud_init_rsa"))

  timeout     = "15m"
  retry_delay = "5s"

  file {
    content =  <<-EOF
touch /tmp/fooisbar
echo Hello World
EOF
    destination = "/tmp/get_k3s_config.sh"
    permissions = "0700"
  }

  commands = [
    "/tmp/get_k3s_config.sh",
    ]
}

output "k3s_coord_server_config_output" {
  value = ssh_resource.k3s_coord_server_config.result
}
```
## TF Output

```shell
ssh_resource.k3s_coord_server_config: Creating...
ssh_resource.k3s_coord_server_config: Creation complete after 0s [id=9206084112850255026]

Apply complete! Resources: 1 added, 0 changed, 1 destroyed.

Outputs:

k3s_coord_server_config_output = ""
```
## Debug Log
```
Created remote file coord-00-7208:22:/tmp/get_k3s_config.sh: 37 bytes
Permissions file /tmp/get_k3s_config.sh:0700:  
command: /tmp/get_k3s_config.sh
done: false
stdout:

stderr:
Run Command Timeout!

error: <nil>
```

**Comment by loafoe** [comment]
@jDmacD thanks for reporting, this is a regression that was introduced recently. The timeout value parsing was changed but the usage was not updated properly, working on a fix now.

**Comment by loafoe** [comment]
@jDmacD should be fixed in `2.6.0`, please try it out.

**Comment by jDmacD** [comment]
@loafoe thats fixed it. Cheers for the fast turn around dude. Great module by the way|
